src/backend/Utils.ts

```diff
@@ -13,7 +13,7 @@ export default class Utils {
     }
 
     const filtered = alerts.filter((alert) => {
-      if (config.hideCancelledWarnings && alert.payload.data.severity === 'Cancel') {
+      if (config.hideCancelledWarnings && (alert.payload.data.severity === 'Cancel' || alert.payload.data.msgType === 'Cancel')) {
```

Comment:
Nice! 👏 I'm pretty sure you found a bug and it's not an API change. Instead of `severity` it should be `msgType`. The `severity` part should be removed in my opinion.